content/blog/2024-01-30-flash-framework.Rmd

```diff
@@ -0,0 +1,69 @@
+
+Here's a quick illustration of why the ranking approach works. The most limited resource in this setting is an expert reviewer's time and attention. We want to show this reviewer as many potential events as possible so that they can interpret the data with their known context. 
+
+<center>
+![**Fig 3.** Illustration of Ranking vs. Alerts for large data streams](/blog/2024-01-30-flash-framework/image1.png)</center>
+
+Now imagine each data point per signal was given an outlier score, and there is an arbitrary binary threshold over all the data. If we develop granular enough rankings, we can directly compare outlier points between different signals without arbitrary thresholds. Then, our data reviewer can focus on data points in their order of importance and stop the review at any time. 
+
+Developing this framework required new methods, visualizations, and evaluations. Over the past few years, our goal as part of the [FlaSH team](https://delphi.cmu.edu/blog/2024/01/01/introducing-the-flash-group/) was to implement and iterate on this framework. We're excited to share insights from these perspectives as we've moved FlaSH from a sandboxed research project to a deployed part of Delphi's systems that helps our users. For the next post in this series, I'll describe some of the novel methods we use to rank events in large volumes of public health data. Then, Richa and Nolan will share their thoughts on moving these research prototypes into a production environment. Third, Tina and Catalina will describe the data visualization and quality assurance processes and considerations they encountered while facilitating event detection investigation. Beyond that, we'll share updates on how we work towards tightening the human-in-the-loop process!
+
+This blog post includes material from my thesis proposal, "Identifying Anomalous Events in Modern
+Public Health Data Streams" 
+
+<a id="#link1">1.</a>  Rosen, George. A history of public health. JHU Press, 2015.  <br>
+<a id="#link2">2.</a>  Chen, Hsinchun, Daniel Zeng, and Ping Yan. Infectious disease informatics: syndromic surveillance for public health and biodefense. Vol. 21. New York: Springer, 2010. <br>
+<a id="#link3">3.</a>  Murphy, Sean Patrick, and Howard Burkom. "Recombinant temporal aberration detection algorithms for enhanced biosurveillance." Journal of the American Medical Informatics Association 15.1 (2008): 77-86. <br>
+<a id="#link4">4.</a> “Rapidly Detecting and Responding to Health Emergencies.” World Health Organization, ]www.who.int/activities/rapidly-detecting-and-responding-to-health-emergencies. Accessed 29 Jan. 2024. <br>
+<a id="#link5">5.</a> Shmueli, Galit, and Howard Burkom. "Statistical challenges facing early outbreak detection in biosurveillance." Technometrics 52.1 (2010): 39-51.
+
```

Comment:
no longer needed, the carat notation for footnotes automatically puts the text at the bottom of the page.
```suggestion

```